src/verifier.cc

```diff
@@ -268,22 +453,30 @@ Result Verifier::Probe(const ProbeCommand* command,
   }
 
   if (count_of_invalid_pixels) {
-    const uint8_t* p = ptr + row_stride * (first_invalid_j + y) +
-                       texel_stride * (x + first_invalid_i);
+    const auto& component = texel_format->GetComponents().back();
+    float scale_factor_for_error_report = 1.0f;
+    if (component.mode == FormatMode::kUNorm ||
+        component.mode == FormatMode::kSNorm ||
+        component.mode == FormatMode::kSRGB) {
+      scale_factor_for_error_report = 255.0f;
```

Comment:
We have the scale factor  for error report here and above why do we need it twice?

Reply:
Removed the above one.